graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Input.java

```diff
@@ -261,6 +264,19 @@ private Long getGaugeValue(String name) {
     public Map<String, Object> getAttributes() {
         return attributes;
     }
+    public Map<String, Object> getAttributes(InputTypeSummaryResponse typeSummaryResponse ) {
+        Map<String, Object> result = Maps.newHashMapWithExpectedSize(attributes.size());
+
+        for (RequestedConfigurationField configurationField : typeSummaryResponse.getRequestedConfiguration()) {
+            if (configurationField.getAttributes().contains("is_password")) {
+                result.put(configurationField.getTitle(), "*******");
```

Comment:
My OCD tells me that this shouldn't be 7 asterisks but 8.
![](http://i.giphy.com/12MISgbheZtmIo.gif)